opentelemetry-otlp/tests/integration_test/README.md

```diff
@@ -15,3 +15,42 @@ popped back out into the files output by the collector.
 * Docker, for the test container
 * TCP/4317 and TCP/4318 free on your local machine. If you are running another
   collector, you'll need to stop it for the tests to run.
+
+## Test Matrix
+
+| **Exporter Type**   | **Provider Creation**     | **Logging**           | **Network Clients**        | **Status** |
```

Comment:
I find it difficult to parse the table for "exhaustiveness". It might be easier if we break it down along some other axis into multiple tables.

Perhaps by network client - as then we'll get 4 tables of 8 rows?